client/asset/eth/eth.go

```diff
@@ -612,17 +613,17 @@ func (eth *ExchangeWallet) lockFunds(amt uint64) error {
 }
 
 // unlockFunds unlocks funds held by the wallet.
-func (eth *ExchangeWallet) unlockFunds(amt uint64) error {
+func (eth *ExchangeWallet) unlockFunds(amt uint64) {
```

Comment:
ACK the removal of the error return since it was being ignored everywhere except `ReturnCoins`, and there's really nothing to be done about the error in most code paths.

And if I understand correctly, we will continue to see an error logged until https://github.com/decred/dcrdex/issues/1466 is resolved not unlocking the refund-reserved funds until all swaps are completed.